src/components/Modal.js

```diff
@@ -218,7 +216,6 @@ class Modal extends React.Component {
                 onKeyUp={e => e.keyCode === 13 && this.props.onRequestClose()}
                 role='button'
                 style={styles.close}
-                tabIndex={0}
```

Comment:
While you're here.... Is the role='button' and onKeyUp really needed on this native button? Can we remove them too?

Reply:
Removed